internal/terraform/terraform_test.go

```diff
@@ -0,0 +1,232 @@
+package terraform
+
+import (
+	"context"
+	"fmt"
+	"github.com/hazelops/ize/internal/config"
+	"github.com/hazelops/ize/pkg/terminal"
+	"io"
+	"os"
+	"testing"
+)
+
+func TestInstall(t *testing.T) {
```

Comment:
Should we include the name of the package in the test name (here and below)? Won't it be unclear which package this is related to when the whole test suite is executed? `TestTerraform_Install`

Reply:
@AutomationD the test of the terraform package and the name of the function is `Install`, not `TerraformIntall`, so everything is fine